spec/support/migration_name_helper.rb

```diff
@@ -0,0 +1,6 @@
+def require_migration
+  spec_name = Gem.location_of_caller.first
+  migration_name = spec_name.sub("spec/migrations", "db/migrate").sub("_spec.rb", ".rb")
+
+  require migration_name
+end
```

Comment:
I changed a few files, very nice with only 1 require up at the top.

But I think I like having the class name in the spec. anyone else have comments here?

Heck, we could even have:

```ruby
def describe_migration(&block)
  spec_name = caller_locations.first.path
  migration_name = spec_name.sub("spec/migrations", "db/migrate").sub("_spec.rb")

  require migration_name
  migration_class = File.basename(migration_name).sub(/^[0-9]+_/, '').constantize
  describe migration_class, &block
end
```

But still has trouble of not defining the class name. but maybe that isn't so important.
Also, don't want to touch const_missing